testhelper/tests.go

```diff
@@ -39,7 +39,10 @@ func (options *TestOptions) TestSetup() {
 // testSetup Setup test
 func (options *TestOptions) testSetup() {
 	if !options.SkipTestSetup {
-		os.Setenv("API_DATA_IS_SENSITIVE", "true")
+
+		if !options.ApiDataIsNonSensitive {
```

Comment:
@toddgiguere, what do you think? I am not a big fan of the double negative, so I would switch to a pointer-based approach. For example:

```go
type TestOptions struct {
    ApiDataIsSensitive *bool
    // ...
}
```

```go
if options.ApiDataIsSensitive == nil {
    // Preserve old behavior
    os.Setenv("API_DATA_IS_SENSITIVE", "true")
} else {
    os.Setenv("API_DATA_IS_SENSITIVE", strconv.FormatBool(*options.ApiDataIsSensitive))
}
```

That way, leaving `ApiDataIsSensitive` as `nil` defaults to sensitive (old behavior), and setting it explicitly to `true` or `false` is straightforward.

Reply:
I agree, a pointer is probably the only way to go here, if it is not a pointer we have no way of determining if the caller didn't set it at all (took the default) or if they really did want it set to `false`. So I agree with the pointer idea.

Let me ask another question though, @ocofaigh I don't know the context of the original ask, but here is my question:
"Is there a legitimate reason why a unit test should EVER want this set to `false`"?

If the answer to the above is "maybe not" then another approach would be to check for the presense of this ENV and if it is not present then just set it to `true`. Don't even have the option to turn off.

Reply:
@Vipin654 please go ahead with the suggested pointer-based implementation thanks